README.md

```diff
@@ -1,70 +1,60 @@
-# Getting Started with Create React App
+# _Pokemon Card Database_
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A database of Pokemon Cards that you can view or add to personal collections.
```

Comment:
this app is not a database, but it does interact with one. You may want to specificy that this is a frontend that connects to an api of Pokemon Cards (which you can link to that repo) that lets you view or add cards to a personal collection